mage_integrations/mage_integrations/tests/sources/api/test_api.py

```diff
@@ -5,19 +5,7 @@
 
 
 def csv_catalog_example():
-    return {'streams':
-            [{'tap_stream_id': 'api', 'replication_method': 'FULL_TABLE', 'key_properties': [],
-             'schema': {'properties': {'column_1': {'type': ['null', 'string']}, 'column_2':
-              {'type': ['null', 'integer']}}, 'type': 'object'},
-              'stream': 'api',
-              'metadata': [{'breadcrumb': (), 'metadata': {'table-key-properties': [],
-                            'forced-replication-method': 'FULL_TABLE',
-                           'inclusion': 'available', 'schema-name': 'api'}},
-                           {'breadcrumb': ('properties', 'column_1'),
-                           'metadata': {'inclusion': 'available'}},
-                           {'breadcrumb': ('properties', 'column_2'),
-                           'metadata': {'inclusion': 'available'}}],
-                'auto_add_new_fields': False, 'unique_conflict_method': 'UPDATE'}]}
+    return {'streams': [{'tap_stream_id': 'api', 'replication_method': 'FULL_TABLE', 'key_properties': [], 'schema': {'properties': {'teste': {'type': ['null', 'integer']}, 'first_name': {'type': ['null', 'string']}, 'second_name': {'type': ['null', 'string']}, 'email': {'type': ['null', 'string']}}, 'type': 'object'}, 'stream': 'api', 'metadata': [{'breadcrumb': (), 'metadata': {'table-key-properties': [], 'forced-replication-method': 'FULL_TABLE', 'inclusion': 'available', 'schema-name': 'api'}}, {'breadcrumb': ('properties', 'teste'), 'metadata': {'inclusion': 'available'}}, {'breadcrumb': ('properties', 'first_name'), 'metadata': {'inclusion': 'available'}}, {'breadcrumb': ('properties', 'second_name'), 'metadata': {'inclusion': 'available'}}, {'breadcrumb': ('properties', 'email'), 'metadata': {'inclusion': 'available'}}], 'auto_add_new_fields': False, 'unique_conflict_method': 'UPDATE'}]} # noqa
```

Comment:
Can you make this dict multi lines?
Each line should have maximum width in flake8 config.

Reply:
changed to multi lines on fcbce12785daa4da685d9ded0d92c35a7df4d5df